app/ide-desktop/lib/client/electron-builder-config.ts

```diff
@@ -101,6 +152,8 @@ const config: Configuration = {
     // TODO [mwu]: Temporarily disabled, signing should be revised.
```

Comment:
Is this TODO affected by this PR?

Reply:
No, the TODO is about macOS code signing, this PR addresses the Windows code signing.